src/style/card.css

```
@@ -0,0 +1,93 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: "Roboto", sans-serif;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  object-fit: contain;
+  object-position: center;
+  margin: 0 auto 40px;
+
+}
+
+.card__title {
+  font-weight: bold;
```

it's better to use digits instead of reserved words. Check the same cases
```suggestion
  font-weight: 700;
```